Packs/MicrosoftDefenderAdvancedThreatProtection/pack_metadata.json

```diff
@@ -17,4 +17,4 @@
         "xsoar",
```

Comment:
Not sure its suppose to be on MPv2
Please validate with @anara123

Reply:
Was added by the find-dependencies comamnd. Not sure why... Will remove this.

Reply:
BTW I saw that it was added within this PR - https://github.com/demisto/content/pull/16553
Any thoughts on why it was added? @David-BMS